### PR TITLE
Remove unnecessary dependency from keywhiz-log

### PR DESCRIPTION
**Pull request by jbpeirce**
This dependency is not in use any longer (it's unclear whether it was ever required)

### CODE REVIEW

File: docs/apidocs/automation_v2-secret-management.json [hunk 1]
```diff
@@ -694,6 +694,13 @@
         "version" : {
           "type" : "integer",
           "format" : "int64"
+        },
+        "contentCreatedAtSeconds" : {
```
Comment (mcpherrinm) [inline docs/apidocs/automation_v2-secret-management.json]: Is this supposed to be included in this change?

Comment (jbpeirce) [inline docs/apidocs/automation_v2-secret-management.json]: It was generated as a side effect when I built Keywhiz to make sure the dependency removal didn't break anything; I could pull it into a separate PR or commit.

Comment (mcpherrinm) [inline docs/apidocs/automation_v2-secret-management.json]: separate commit is fine

Comment (jbpeirce) [inline docs/apidocs/automation_v2-secret-management.json]: Done.
